httpsec/client_ip.go

```diff
@@ -0,0 +1,126 @@
+package httpsec
+
+import (
+	"net"
+	"net/textproto"
+	"strings"
+
+	"github.com/DataDog/appsec-internal-go/netip"
+)
+
+const (
+	// RemoteIPTag is the tag name used for the remote HTTP request IP address.
+	RemoteIPTag = "network.client.ip"
+	// ClientIPTag is the tag name used for the client IP deduced from the HTTP
+	// request headers with ClientIP().
+	ClientIPTag = "http.client_ip"
+)
+
+// ClientIPTags returns the resulting Datadog span tags `http.client_ip`
+// containing the client IP and `network.client.ip` containing the remote IP.
+// The tags are present only if a valid ip address has been returned by
+// ClientIP().
+func ClientIPTags(remoteIP, clientIP netip.Addr) (tags map[string]string) {
+	remoteIPValid := remoteIP.IsValid()
+	clientIPValid := clientIP.IsValid()
+	if !remoteIPValid && !clientIPValid {
+		return nil
+	}
+
+	tags = make(map[string]string, 2)
+	if remoteIPValid {
+		tags["network.client.ip"] = remoteIP.String()
+	}
+	if clientIPValid {
+		tags["http.client_ip"] = clientIP.String()
```

Comment:
Same with `ClientIPTag`
```suggestion
		tags[ClientIPTag] = clientIP.String()
```